src/QueryType/Ping/Query.php

```diff
@@ -31,7 +31,7 @@ class Query extends BaseQuery
     protected $options = [
         'resultclass' => Result::class,
         'handler' => 'admin/ping',
-        'omitheader' => true,
+        'omitheader' => false,
```

Comment:
Could you explain that change?

Reply:
Ping result used to have this hardcoded:

```php
    public function getStatus(): int
    {
        return 0;
    }
```

With my changes, calling `$result->getStatus()` on a ping calls the generic implementation in `Core\Query\Result\QueryType`. To maintain BC and still have it return `0` on success, the responseHeader must be included in the response.

Reply:
I included this in the CHANGELOG and mentioned it in the docs.